projects/js-packages/components/components/icon-tooltip/index.tsx

```diff
@@ -36,6 +36,8 @@ const IconTooltip: React.FC< IconTooltipProps > = ( {
 	offset = 10,
 	title,
 	children,
+	shadowAnchor = false,
```

Comment:
I understand we explain this property in a code comment within our type file, but I find this property name unintuitive. Could we use a more descriptive name, like `popoverWrapperStyle`?

Reply:
The naming was really bothering me. However, I got an idea to name it `popoverAnchorStyle` with `icon` and `wrapper` values, which will also benefit the future refactoring to align with the [Popover component prop](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/popover#anchor-element--virtualelement--null).